config/resources/lpdc.json

```diff
@@ -0,0 +1,644 @@
+        },
+        "type": {
+          "type": "string",
+          "predicate": "schema:contactType"
+        },
+        "opening-hours": {
+          "type": "string",
+          "predicate": "schema:openingHours"
+        }
+      },
+      "relationships": {
+        "hasContactPoint": {
+          "target": "conceptual-public-services",
+          "predicate": "m8g:hasContactPoint",
+          "cardinality": "many",
+          "inverse": true
+        }
+      },
+      "new-resource-base": "http://data.lblod.info/id/contact-punten/"
+    },
+    "documents": {
+      "class": "dcat:Distribution",
+      "attributes": {
+        "description": {
+          "type": "string",
+          "predicate": "dct:description"
+        },
+        "identifier": {
+          "type": "string",
+          "predicate": "dct:identifier"
+        },
+        "name": {
+          "type": "string",
+          "predicate": "dct:title"
+        }
+      },
+      "relationships": {
+        "hasAttachment": {
+          "target": "conceptual-public-services",
+          "predicate": "lpdcExt:attachment",
+          "cardinality": "many",
+          "inverse": true
+        }
+      },
+      "features": [
+        "include-uri"
+      ],
+      "new-resource-base": "http://data.lblod.info/id/requirement/"
+    },
+    "locations": {
+      "class": "dct:Location",
+      "relationships": {
+        "hasAttachment": {
+          "target": "conceptual-public-services",
+          "predicate": "dct:spatial",
+          "cardinality": "many",
+          "inverse": true
+        }
+      },
+      "features": [
+        "include-uri"
+      ],
+      "new-resource-base": "http://data.lblod.info/id/location/"
+    },
+    "lifecycle-statuses": {
+      "class": "code:LifecycleStatus",
+      "attributes": {
+        "label": {
+          "type": "string",
+          "predicate": "skos:prefLabel"
+        }
+      },
+      "new-resource-base": "http://lblod.data.gift/concepts/"
+    },
+    "sectors": {
+      "TODO" : "NACE codes? cfr dienstencataloog AP",
+      "class": "code:Sector",
+      "attributes": {
+        "label": {
+          "type": "string",
+          "predicate": "skos:prefLabel"
+        }
+      },
+      "new-resource-base": "http://lblod.data.gift/concepts/"
+    },
+    "target-audiences": {
+      "class": "code:TargetAudience",
+      "attributes": {
+        "label": {
+          "type": "string",
+          "predicate": "skos:prefLabel"
+        }
+      },
+      "new-resource-base": "http://lblod.data.gift/concepts/"
+    },
+    "thematic-areas": {
+      "class": "code:ThematicArea",
+      "attributes": {
+        "label": {
+          "type": "string",
+          "predicate": "skos:prefLabel"
+        }
+      },
+      "new-resource-base": "http://lblod.data.gift/concepts/"
+    },
+    "service-types": {
+      "TODO": "COFOG Code list? cfr dienstencataloog AP",
+      "class": "code:ServiceType",
```

Comment:
It seems this predicate isn't the one that's used in the migration files, so it doesn't return any data.

`http://lblod.data.gift/vocabularies/lpdc-ipdc/IPDCServiceType`

Reply:
Correct, fixed it, also changed other codelists.
ToDo is about linking this to an existing external code list (beyond PoC)